### PR TITLE
added new theme 'compact.css'

### PR DESCRIPTION
**Pull request by stevkazt**
Hey, thanks for working on this browser I'd like to contribute and I think a good start is a theme, I made it few days ago for me and I think someone else may like it, I moved navbar and tabs to the same line so you can get even more space for page view and focus on keybindings usage, icon and mode space are smaller.

[Check a screenshot here](https://i.imgur.com/31AWR1Q.png)

### CODE REVIEW

File: app/colors/compact.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+/*
+ * APP
+*/
+/* navbar & tabs */
+#navbar {width: 40vw; height: 1.8em;}
+#logo {height: 1.2em;width: 1.2em;min-width: 1.2em;min-height: 1.2em; padding:0.2em;}
+#mode-container {width: 6em;}
+#mode {font-size: 1.1em;}
+
```
Comment (Jelmerro) [inline app/colors/compact.css:14]: I don't really think this creates extra space, it only changes the sizes slightly, but the margins remain. Removing lines 11-13 will make the theme simpler, and easier to understand, while at the same time maintaining the default look of Vieb better. What do you think?

**Review by Jelmerro — COMMENTED** [review]
I like this idea of a single bar combining the navbar and the tabs, it looks pretty good.

File: app/colors/compact.css (hunk 1 above)
Comment (stevkazt) [reply app/colors/compact.css]: Perfect I just did a couple of changes to keep the default look, it is still good for me and happy if we can share it.

Comment (Jelmerro) [reply app/colors/compact.css]: Thanks, looks good now!

**Pull request merged**